src/main/java/seedu/guest/logic/commands/MarkRoomUncleanCommand.java

```diff
@@ -0,0 +1,68 @@
+package seedu.guest.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.guest.model.Model.PREDICATE_SHOW_ALL_GUESTS;
+
+import java.util.List;
+
+import seedu.guest.logic.commands.exceptions.CommandException;
+import seedu.guest.model.Model;
+import seedu.guest.model.guest.Bill;
+import seedu.guest.model.guest.DateRange;
+import seedu.guest.model.guest.Email;
+import seedu.guest.model.guest.Guest;
+import seedu.guest.model.guest.IsRoomClean;
+import seedu.guest.model.guest.Name;
+import seedu.guest.model.guest.NumberOfGuests;
+import seedu.guest.model.guest.Phone;
+
+
+/**
+ * Changes isRoomClean field of all guest in current list to "no".
+ */
+public class MarkRoomUncleanCommand extends Command {
+
+    public static final String COMMAND_WORD = "markRoomUnclean";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Changes all guest RoomClean statuses to \"no\".\n"
```

Comment:
I think this could be either `all guests' room clean statuses` or `all guests' isRoomClean statuses`